<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserViewWindow</class>
 <widget class="QMainWindow" name="UserViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>User View</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>800</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>600</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="name_label">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
background-color: white; 
color: black; 
font-size: 20px;</string>
        </property>
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLineEdit" name="name_filter">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
background-color: white; 
color: black; 
font-size: 20px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="past_view_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
border-radius: 14px;
background-color:rgb(207, 67, 74); 
color: white; 
letter-spacing: 0.7px; 
font-size: 25px;</string>
        </property>
        <property name="text">
         <string>Past View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="admin_view_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
border-radius: 14px;
background-color:rgb(207, 67, 74); 
color: white; 
letter-spacing: 0.7px; 
font-size: 25px;</string>
        </property>
        <property name="text">
         <string>Admin View</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="rightMargin">
       <number>550</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="department_label">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
background-color: white; 
color: black; 
font-size: 20px;</string>
        </property>
        <property name="text">
         <string>Department: </string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLineEdit" name="department_filter">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
background-color: white; 
color: black; 
font-size: 20px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="current_week_time_table">
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    color: black;
    gridline-color: black;
    border: 2px solid black;  /* Outer border of the whole table */
    font: 10px;
	font-size: 12px;
}
QTableView::item {
    border-top: 1px solid black;  /* Borders for each cell */
    border-bottom: 1px solid black;
    border-left: 1px solid black;
    border-right: 1px solid #555;
}
QTableCornerButton::section {
    background-color: rgb(78, 149, 213);
}
QTableView::item:selected {
    background-color: rgb(144, 144, 144);  /* Background color for selected item */
    color: black;  /* Text color for selected item */
}
QTableView::item:focus {
    border: 2px solid rgb(242, 128, 133);
    background-color: rgb(144, 144, 144);
}
QHeaderView::section {
    background-color: rgb(78, 149, 213);
    color: white;
    height: 35px;
    font: 14px;
}
QScrollBar:vertical {
    background: rgb(188, 224, 235);
}
QScrollBar::handle:vertical {
    background: rgb(71, 153, 176);
}
QScrollBar:horizontal {
    background: rgb(188, 224, 235);
}
QScrollBar::handle:horizontal {
    background: rgb(71, 153, 176);
}
QLineEdit {
    color: black;  /* Text color for QLineEdit (editor) */
}</string>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/IMGs/subbaroo_long_name.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="day_toggle_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
border-radius: 14px;
background-color: rgb(78, 149, 213); 
color: white; 
letter-spacing: 1px; 
font-size: 25px;</string>
        </property>
        <property name="text">
         <string>Day Toggle: OFF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <fontweight>ExtraBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
border-radius: 14px;
background-color: rgb(78, 149, 213); 
color: white; 
letter-spacing: 1px; 
font-size: 25px;</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>400</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QLabel" name="current_week_icon">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/IMGs/current_week.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QLabel" name="current_week_label">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555; 
background-color: rgb(133, 193, 111);
color: white; 
letter-spacing: 0.6px; 
font-size: 25px;
</string>
        </property>
        <property name="text">
         <string>  Current Week</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
